tools/jvm/src/main/scala/org/scalajs/core/tools/linker/frontend/LinkerFrontendPlatformExtensions.scala

```diff
@@ -0,0 +1,27 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js tools             **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2017, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+package org.scalajs.core.tools.linker.frontend
+
+import org.scalajs.core.tools.linker.frontend.optimizer._
+
+private[frontend] object LinkerFrontendPlatformExtensions {
```

Comment:
Yes, indeed. `Extensions` are ultimately part of the public API because they are inherited. Whereas this thing is an implementation detail that need never be seen.